failsafe/failsafe.py

```diff
@@ -75,7 +77,9 @@ def __init__(self, retry_policy=None, circuit_breaker=None):
             except Exception as e:
                 context.errors += 1
                 recent_exception = e
-                retry = self.retry_policy.should_retry(context, e)
+                retry = self.exception_handling_policy.should_retry(context, e)
+                if self.exception_handling_policy.should_raise(e):
```

Comment:
Ah, good point!

I think that having exception as both retriable and abortable doesn't make sense for the user. Or does it? What would be the use case? I suggest adding logic to constructor of `RetryPolicy` which will protect from having the same exception in both lists, what do you think? That way we will have very clearly defined behaviour.